mailer.rst

```diff
@@ -129,6 +129,13 @@ Service               Install with
     <mail-catcher>` instead. Note that most supported providers also offer a
     free tier.
 
+.. note::
+
+    When using a 3rd party transport, you might want to use a web API
+    instead of directly rely on the SMTP protocol to send emails.
```

Comment:
This sounds a bit like we preferred web APIs over SMTP. I had something more neutral in mind, like…

```suggestion
    Besides SMTP, many 3rd party transports offer a web API to send emails.
```